From: Antonio Terceiro <terceiro@debian.org>
Date: Thu, 3 Mar 2016 16:30:17 -0300
Subject: Be careful with that bundler

--- a/railties/lib/rails/generators/app_base.rb
+++ b/railties/lib/rails/generators/app_base.rb
@@ -409,20 +409,15 @@
         # things in the same process is a recipe for a night with paracetamol.
         #
         # Thanks to James Tucker for the Gem tricks involved in this call.
-        _bundle_command = Gem.bin_path("bundler", "bundle")
 
         require "bundler"
         Bundler.with_original_env do
-          exec_bundle_command(_bundle_command, command)
-        end
-      end
-
-      def exec_bundle_command(bundle_command, command)
-        full_command = %Q["#{Gem.ruby}" "#{bundle_command}" #{command}]
-        if options[:quiet]
-          system(full_command, out: File::NULL)
-        else
-          system(full_command)
+          full_command = %Q["#{Gem.ruby}" "/usr/bin/bundle" #{command}]
+          if options[:quiet]
+            system(full_command, out: File::NULL)
+          else
+            system(full_command)
+          end
         end
       end
 
@@ -451,7 +446,7 @@
       end
 
       def run_bundle
-        bundle_command("install") if bundle_install?
+        bundle_command("install --local") if bundle_install?
       end
 
       def run_webpack
